index.html

```diff
@@ -201,7 +201,43 @@ <h3 class="about_sub_title">
           />
         </h3>
       </ul>
-    </section>    
+    </section>        
+    <footer class="contact">
+      <div class="contact-us">
+        <div class="contact-text">
+          <h2 class="form-heading">Contact me</h2>
+          <p class="my-info">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d
+            love to help with it
+          </p>
+          <form
+            class="contact-form"
+            action="https://formspree.io/f/mbjbpglg"
+            method="post"
+          >
+            <label for="your-name">
+              <input type="text" id="name" placeholder="John Smith" name="name" maxlength="30" required/>
+            </label>
+
+            <label for="your email">
+              <input type="email" id="email-id" placeholder="example@gmail.com" name="email-id" required/>
+            </label>
+
+            <label for="your message">
+              <textarea placeholder="Write your message here" maxlength="500" id="message" required></textarea>
+            </label>
+
+            <button type="submit" class="submit-button">
+              Get in touch
+            </button>
```

Comment:
### Your Footer
![Capture](https://user-images.githubusercontent.com/93039359/199463752-21187b35-f3fe-4d80-89db-1beeffe9eaa3.PNG)

### Figma Footer
![Capture3Figma](https://user-images.githubusercontent.com/93039359/199463788-b0991161-9190-4920-a562-74b8f14ab14f.PNG)

- [ ] Your footer is missing those images which you can see in the Figma image above ⬆️ Kindly see that picture and add those missing images in your design to make it look similar to the Figma template 👍